includes/hooks/template.php

```diff
@@ -149,6 +149,10 @@
 // Add the quiz pagination.
 add_action( 'sensei_single_quiz_questions_after', array( 'Sensei_Quiz', 'the_quiz_pagination' ), 9, 0 );
 
+// @since 3.15.0
+// Add the quiz progress bar.
+add_action( 'sensei_single_quiz_questions_before', array( 'Sensei_Quiz', 'the_quiz_progress_bar' ), 20 );
```

Comment:
I think you should move this action next to the other `sensei_single_quiz_questions_before` actions (a few lines up).

Reply:
https://github.com/Automattic/sensei/pull/4606/commits/4e25e04e019978f6f6c19e6b99fc04e5e9aa17a6#diff-0ecde7636e507a880669f45547c52d627c717b9c8f1b3dc38be7635c50e1e9c3R134